README.md

```diff
@@ -565,3 +565,75 @@ const results = await testSheet.testRecords(inputRows)
+
+Steps
+- Clone the platform-sdk-starter
+```bash
+git clone git@github.com:FlatFilers/platform-sdk-starter.git
+```
+- Create your own private repo
+Navigate to https://github.com/new
+Fill out details as in the picture, take care to make the repo private and not check any other boxes.  The example repo name for this tutorial will be `private-platform-sdk-starter` and it lives at https://github.com/paddymul/private-platform-sdk-starter (check that your new repo isn't accessible from a non-logged in private browsing tab before proceeding)
+
+![Create a private repo](/assets/GH-create-private-repo.png)
+- Push to your new private repo
+``` 
+cd platform-sdk-starter
+git remote add gh-private git@github.com:paddymul/private-platform-sdk-starter.git  #replace with your own repo
+git remote remove origin # remove the original origin so we don't accidentally push there
+git push gh-private main     # note gh-private
+```
+- Create your implementation branch
+Creating a separate branch makes for a more readable PR for collaboration
+```bash
+git checkout -b feat/my-implmentation   #create your branch
+echo >> src/index.ts    #add a newline to the end of the file for a change
+git commit -am "my first commit to feat/my-implmentation"
+git push gh-private feat/my-implmentation  #push your code to your private github repo
+```
+The last line will display the following info
+```bash
+
+~/platform-sdk-starter $ git push gh-private feat/my-implmentation
+Enumerating objects: 7, done.
+Counting objects: 100% (7/7), done.
+Delta compression using up to 10 threads
+Compressing objects: 100% (4/4), done.
+Writing objects: 100% (4/4), 420 bytes | 420.00 KiB/s, done.
+Total 4 (delta 2), reused 0 (delta 0), pack-reused 0
+remote: Resolving deltas: 100% (2/2), completed with 2 local objects.        
+remote: 
+remote: Create a pull request for 'feat/my-implmentation' on GitHub by visiting:        
+remote:      https://github.com/paddymul/private-platform-sdk-starter/pull/new/feat/my-implmentation        
+remote: 
+To github.com:paddymul/private-platform-sdk-starter.git
+ * [new branch]      feat/my-implmentation -> feat/my-implmentation
+
+```
+- Create your PR
+Go to the URL suggested from the command line
+https://github.com/paddymul/private-platform-sdk-starter/pull/new/feat/my-implmentation
+![Create a private repo](/assets/GH-create-PR.png)
+Click create PR
+
+- Invite flatfile collaborators
+This step allows us to see the code you have written to the repo
+Navigate to the settings for your new repo
+
+https://github.com/paddymul/private-platform-sdk-starter/settings
+Then navigate to the access pane 
+https://github.com/paddymul/private-platform-sdk-starter/settings/access
+at this point github will ask for two factor auth, fill it out.
+![Add a collaborator](/assets/GH-Add-a-collaborator-Rock.png)
+
+Add the names of `bangarang` and `paddymul`
+
+- Notify Flatfile that you shared the PR with us
```

Comment:
❔ How should they notify FF (outside of a GitHub notification)? Email Support? Not everyone will have their own implementation team

Reply:
> Once you've added collaborators, please notify Flatfile Support. If you have an ongoing ticket with our Success or Support teams, you can update the ticket to let them know that you have shared your PR with our team. If you do not yet have an open ticket, email support@flatfile.com with a description of your problem and include a note to let us know that you have already shared your PR.